ui2/src/claims.ts

```diff
@@ -30,8 +49,14 @@ export const transformClaim = (claim : Claim<Date, Id>, linkText : string) : any
   switch (claim.tag) {
     case "When":
       return { ...claim, linkText, type: "Claim", children: [ transformObservation(claim.value.predicate, "condition"), transformClaim(claim.value.obligation, "then") ] };
+    case "Scale":
+      return { ...claim, linkText, type: "Claim", children: [ transformObservation(claim.value.k, "factor"), transformClaim(claim.value.obligation, "then") ] };
+//  case "Give": //FIXME: this causes the switch to fail for some weird reason
+//    return { ...claim, linkText, type: "Claim", children: [ transformClaim(claim.value, "claim") ] };
```

Comment:
@GeorgSchneider can you have a look at why this is failing to compile? It would be nice to have this so the ADT is complete.